themes/codeorigin.jquery.com/page.php

```diff
@@ -1,7 +1,54 @@
+<?php
+
+// attach custom header and footer tags
+function header_append() {
+	echo '
+	<link rel="stylesheet" href="' . get_template_directory_uri() . '/css/font-awesome.css" />
```

Comment:
I saw it in the themes/jquery/css directory, but not included. I'l drop this out.